D1673/blas_interface.md

```diff
@@ -18,7 +18,7 @@
 ## Contributors
 
 * Chip Freitag (chip.freitag@amd.com) (AMD)
-* Bryce Lelbach (blelbach@nvidia.com) (NVIDIA)
+* Bryce Adelstein Lelbach (brycelelbach@gmail.com) (NVIDIA)
 * Srinath Vadlamani (Srinath.Vadlamani@arm.com) (ARM)
 * Rene Vanoostrum (Rene.Vanoostrum@amd.com) (AMD)
 
```

Comment:
Could you update my contact info to dansunderland@gmail.com?

Reply:
Hi @dsunder ! : - )  I'll be happy to update your contact info.  I'll need to make another pass in any case to incorporate the P2299 changes (esp. `basic_mdspan` -> `mdspan`), so I can update the P0009 version then.

Reply:
@dsunder PR https://github.com/ORNL/cpp-proposals-pub/pull/144 updates your contact info -- please feel free to review if you like -- thanks! : - )